universal-application-tool-0.0.1/app/repository/QuestionRepository.java

```diff
@@ -33,6 +33,57 @@ public QuestionRepository(EbeanConfig ebeanConfig, DatabaseExecutionContext exec
         executionContext);
   }
 
+  static class PathConflictDetector {
+    Boolean conflict = Boolean.FALSE;
+    String newPath;
+
+    public PathConflictDetector(String newPath) {
```

Comment:
Question (blocking): does this need to be public? the class itself is package-private and I don't see it accessed outside of this package.

Reply:
Sure thing. Will change it.

Reply:
Done.